app/components/user-profile-permissions.js

```diff
@@ -31,99 +32,84 @@ export default class UserProfilePermissionsComponent extends Component {
   @tracked instructedSessions = [];
   @tracked studentAdvisedSessions = [];
 
+  get courseCount() {
+    return (
+      this.directedCourses.length +
+      this.administeredCourses.length +
+      this.instructedCourses.length +
+      this.studentAdvisedCourses.length
+    );
+  }
+
+  get sessionCount() {
+    return (
+      this.administeredSessions.length +
+      this.instructedSessions.length +
+      this.studentAdvisedSessions.length
+    );
+  }
+
   @restartableTask
   *load() {
     this.academicYearCrossesCalendarYearBoundaries = yield this.iliosConfig.itemFromConfig(
       'academicYearCrossesCalendarYearBoundaries'
     );
 
-    const map = yield hash({
-      schools: this.store.findAll('school'),
-      academicYears: this.store.findAll('academic-year'),
-      defaultSchool: this.args.user.school,
-    });
-
-    this.schools = map.schools;
-    this.academicYears = map.academicYears;
+    this.schools = (yield this.store.findAll('school', { reload: true })).toArray();
```

Comment:
Why the reload?

Reply:
good question - beats me - it yields an empty collection without.

Reply:
We preload schools in the application controller to ensure all subsequent loads are fast. Is it empty only in tests? Or when used in the app? I'll investigate.

Reply:
check out `app/adapters/school.js`, i'm fairly certain that the defined loading behaviour there is causing the problem here. forcing a reload here, as an exception to the rule, seems acceptable to me.

Reply:
Ok, looks like I broke this earlier this year and I'm surprised it hasn't bitten us before this. Several places in the app only work now because users usually navigate directly from the dashboard or else never change from their default school. I added a fix in https://github.com/ilios/frontend/pull/6235 and once that is merged this `reload` can be removed.

Reply:
:heavy_check_mark: